tests/features/account/account_delete.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Account Delete
+  As a user,
+  I want to delete my account and all the data.
+
+  Scenario: User Rincewind visits the delete endpoint
+    Given user Rincewind is logged in
+    When user Rincewind visits the delete endpoint
+    Then user Rincewind is prompter for delete confirmation
+
+  Scenario: User Ridcully visits the delete endpoint
+    Given user Ridcully is logged in
+    And user Ridcully is the sole owner of Unseen University
+    When user Ridcully visits the delete endpoint
+    Then 'This account has organizations without co-owners' warning is shown to the user
```

Comment:
Replace hardcoded strings with explanatory text in all scenarios. This one:

```gherkin
Then delete is blocked because the account is sole owner of some organizations
```